src/kibana/plugins/settings/sections/objects/_objects.js

```diff
@@ -12,42 +16,52 @@ define(function (require) {
   .directive('kbnSettingsObjects', function (config, Notifier, Private, kbnUrl) {
     return {
       restrict: 'E',
-      controller: function ($scope, $injector, $q, AppState) {
+      controller: function ($scope, $injector, $q, AppState, es) {
+        var notify = new Notifier({ location: 'Saved Objects' });
 
         var $state = $scope.state = new AppState();
-
-        var resetCheckBoxes = function () {
-          $scope.deleteAll = false;
-          _.each($scope.services, function (service) {
-            _.each(service.data, function (item) {
-              item.checked = false;
-            });
-          });
-        };
+        $scope.currentTab = null;
+        $scope.selectedItems = [];
 
         var getData = function (filter) {
```

Comment:
You're right, `data` is almost always the hits that come back per registered service (i.e. dashboards, searches, visualizations).

`$q.all(services).then(function (data) {`

Here, `data` could better be renamed something like you've said, maybe `tab` or `section` (or, in other places in this same code, it's referred to as `service`, which is also confusing). It's what's being returned from the preceding function (`{service: service, serviceName: ...}`).

I believe that's the only place `data` is referring to something other than `hits`.